hydroserving/cli/commands/root.py

```diff
@@ -50,9 +50,12 @@
               required=False,
               default=False,
               is_flag=True)
+@click.option('--monitoring_configuration',
```

Comment:
how does user pass the config here?

Reply:
What you are saying is that it should not be passed via click, only parsed?

Reply:
Deleted